.travis.yml

```diff
@@ -1,8 +1,22 @@
 language: rust
+os:
+    - linux
+    - osx
 rust:
-- stable
-- beta
+    - stable
+    - beta
+matrix:
+    allow_failures: beta
+    fast_finish: true
+before_script:
+    - cargo uninstall rustfmt || true
```

Comment:
do we really need this? seems not
https://travis-ci.org/golemfactory/golem-client/jobs/535998976#L109
https://travis-ci.org/golemfactory/golem-client/jobs/535998977#L109

Reply:
Good question. At the moment, as you aptly pointed out, we don’t really need it. It was there so that if there was an old `rustfmt` lying somewhere in travis, we’d able to install a new version of it (it’d error out if a version already existed in the OS). But, that’s probably a rather unlikely scenario indeed, so I’m happy to remove this line :-)